pyproject.toml

```diff
@@ -23,6 +23,7 @@ appdirs = "~1.4"
 tweepy = "~4.1"
 facebook-sdk = {git = "https://github.com/mobolic/facebook-sdk.git"}
 aerich = "~0.6"
+python-telegram-bot = "^13.10"
```

Comment:
What about using `~13.10` ? I believe it allows for semver upgrades. We'd still lock dependencies with `poetry.lock` and `channels-lock.scm`

Reply:
~ upgrades until the minor changes, ^ upgrades until the major changes. I don't think it makes a big difference for this package